src/modules/hubspot/repository/hubspotRepository.ts

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import crypto from 'crypto';
+import { URLSearchParams } from 'url';
+import authRepository from '../../auth/repository/authRepository';
+import { PLATFORM, USER_ID } from '../../../utils/variablesUtils';
+
+const handleHubspotAuth = async (auth, user_id) => {
+    const data  = new URLSearchParams({
+        ...auth,
+        client_id: process.env.HUBSPOT_CLIENT_ID,
+        client_secret: process.env.HUBSPOT_CLIENT_SECRET,
+        redirect_uri: process.env.HUBSPOT_AUTH_REDIRECT_URL,
+    });
+
+    const response = await axios.post('https://api.hubapi.com/oauth/v1/token', data, { headers: { 'Content-Type': 'application/x-www-form-urlencoded' }});
```

Comment:
Instead of axios, create a fetch api helper with fixed parameters that are commonly used and onli pass the needed info as props.